main.rb

```diff
@@ -1,10 +1,51 @@
-require_relative 'lib/person'
 require_relative 'lib/capitalize_decorator'
 require_relative 'lib/trimmer_decorator'
+require_relative 'lib/classroom'
+require_relative 'lib/student'
+require_relative 'lib/person'
+require_relative 'lib/rental'
+require_relative 'lib/book'
 
 person = Person.new(22, 'maximilianus')
 p person.correct_name
 capitalized_person = CapitalizeDecorator.new(person)
 p capitalized_person.correct_name
 capitalized_person = TrimmerDecorator.new(capitalized_person)
 p capitalized_person.correct_name
+
+puts '-------------------------------------------------------------------------------------------------'
+
+classroom1 = Classroom.new('Year_1_computer_science')
+
+jenny = Student.new(23, nil, 'Jenny')
+james = Student.new(19, nil, 'James')
+john = Student.new(20, nil, 'John')
+
+# jenny = Student.new(23, classroom1, 'Jenny')
+# james = Student.new(19, classroom1, 'James')
+# john = Student.new(20, classroom1, 'John')
+
```

Comment:
- [ ] Good job :100: , I really appreciate your code :+1:  :+1:. So, I noticed that you push your project with some commented code lines. I think It will be better to remove them in order to make your code cleaner :+1:  and it will be completed. Happy coding.

Reply:
Hey @andersonlebon 

Thank You for the review :pray: 
I really appreciate :+1: :+1: